python/chigger/base/KeyPressInteractorStyle.py

```diff
@@ -32,4 +32,5 @@ def keyPress(self, obj, event): #pylint: disable=unused-argument
         """
         key = obj.GetInteractor().GetKeySym()
         if key == 'c':
-            print '\n'.join(utils.print_camera(self.GetDefaultRenderer().GetActiveCamera()))
+
+            print '\n'.join(utils.print_camera(self.GetCurrentRenderer().GetActiveCamera()))
```

Comment:
I don't see the diff here (other than the extra blank line) Am I missing something? This supposedly fixes the camera?

Reply:
self.GetDefaultRenderer() -> self.GetCurrentRenderer()